### PR TITLE
DOCKER-365 use update 10 base image

### PR DESCRIPTION
**Pull request by RVanhuysseXenit**

### CODE REVIEW

**Comment by RVanhuysseXenit** [comment]
Still need to check why the build failed

**Comment by RVanhuysseXenit** [comment]
> Still need to check why the build failed

Resolved: we switched to release and snapshot versions of docker images. The release version of the base image was not yet available

File: tomcat-8.5/jdk-11-ubuntu-18.04.gradle [hunk 1]
```diff
@@ -11,7 +11,7 @@ ext {
             flavor : 'jdk',
             version: [
                     major: 11,
-                    update: 9
+                    update: 10
```
Comment (WimCrols) [inline tomcat-8.5/jdk-11-ubuntu-18.04.gradle:14]: minor does not need to be set? I see it was't present before either, which is unexpected.

**Review by WimCrols — CHANGES_REQUESTED** [review]

File: tomcat-8.5/jdk-11-ubuntu-18.04.gradle (hunk 1 above)
Comment (WimCrols) [reply tomcat-8.5/jdk-11-ubuntu-18.04.gradle:14]: OK, I checked other files and it isn't set anywhere, so you can merge.

**Review by WimCrols — APPROVED** [review]

File: tomcat-8.5/jdk-11-ubuntu-18.04.gradle (hunk 1 above)
Comment (RVanhuysseXenit) [reply tomcat-8.5/jdk-11-ubuntu-18.04.gradle:14]: minor version is hardcoded it seems. As long as we dont need it, I suppose we can leave it as is.

**Pull request merged**